deployment/hasura/migrations/AerieUI/2_extensions/up.sql

```diff
@@ -0,0 +1,56 @@
+create table extensions (
+  description text,
+  id integer generated always as identity,
```

Comment:
Very minor, can ignore: can the PK please be moved to the first line in this `create table` statement?